Makefile

```diff
@@ -1,6 +1,7 @@
 
-GORELEASER_VERSION=v0.141.0
-GORELEASER_CMD=curl -sL https://git.io/goreleaser | GOPATH=$(mktemp -d) VERSION=$(GORELEASER_VERSION) bash -s -- --rm-dist
```

Comment:
Looks like this script is deprecated— the options for getting a pre-built binary are 1. use a package manager (which would make this less portable between Linux & Mac) or 2. grab it from the GitHub releases using the typical "filename contains platform" pattern.

Reply:
[This page](https://goreleaser.com/install/) wrongly claims that you can use `go install` for this purpose— you can certainly get the tool that way, but it would be building it from source.